src/comm/bench-flow.js

```diff
@@ -278,6 +280,8 @@ function defaultTest(args, clientArgs, final) {
  * @param {String} networkFile path of the blockchain configuration file
  */
 module.exports.run = function(configFile, networkFile) {
+    let localConfig = require(configFile);
+    configurationType = localConfig.test.clients.WITH_MQ;
```

Comment:
localConfig's value is same as config in line 290. configurationType  can be got after the config is set.